docs/data-sources/associated_site.md

```diff
@@ -0,0 +1,72 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Multi-Site"
+layout: "aci"
+page_title: "ACI: aci_associated_site"
+sidebar_current: "docs-aci-data-source-aci_associated_site"
+description: |-
+  Data source for Associated Site
+---
+
+# aci_associated_site #
+
+Data source for Associated Site
+
+## API Information ##
+
+* Class: [fvSiteAssociated](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvSiteAssociated/overview)
+
+
+* Distinguished Name Format: `uni/tn-{name}/ctx-{name}/stAsc`
```

Comment:
is this not supporting more dn_formats? please add to the classes.yml those dn_formats that should be added